ddl/ddl_api.go

```diff
@@ -4274,40 +4254,30 @@ func checkColumnWithIndexConstraint(tbInfo *model.TableInfo, originalCol, newCol
 	}
 
 	pkIndex := tables.FindPrimaryIndex(tbInfo)
-	var clusteredPkLen int
-	if tbInfo.IsCommonHandle {
-		var err error
-		clusteredPkLen, err = indexColumnsLen(columns, pkIndex.Columns)
-		if err != nil {
-			return err
-		}
-	}
 
-	checkOneIndex := func(indexInfo *model.IndexInfo, pkLenAppendToKey int, skipCheckIfNotModify bool) (modified bool, err error) {
```

Comment:
Now, it seems the return value `modified` is not used. Can it be removed?

Reply:
yes， I think so.